pytorch_lightning/trainer/logging.py

```diff
@@ -34,7 +34,13 @@ def configure_logger(self, logger):
         elif logger is False:
             self.logger = None
         else:
-            self.logger = logger
+            try:
+                _ = iter(logger)
```

Comment:
this looks too complicated... shall it be rather `isinatance(obj, (list, tuple, ...))` because with iter you may skip the first one

Reply:
We could potentially do `isinstance(obj, abc.Iterable)` but this will still fail in some cases - see the discussion [here](https://stackoverflow.com/a/1952655). Since we use `isinstance` elsewhere in the codebase I don't mind changing it for consistency - but I would opt for `isinstance(obj, abc.Iterable)` rather than checking directly if it's a list / tuple

Reply:
Done :)

Reply:
I did something similar here, have look if you find it useful...
https://birl.readthedocs.io/en/latest/birl.utilities.experiments.html#birl.utilities.experiments.is_iterable